MAESTRO/utils/scATAC_FragmentGenerate.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
this scATAC_FragmentGenerate.py  script should be under MAESTRO/ folder. https://github.com/liulab-dfci/MAESTRO/tree/master/MAESTRO I forgot why I have it in the utils folder. in the Snakefile, we should use the one in the MAESTRO/folder rather than using this.